### PR TITLE
update: API3 docs 

### PR DESCRIPTION
**Pull request by vanshwassan**
### Description

- Updated the API3 docs under `/builders/integrations/oracles/api3.md` with the new [API3 Market](https://market.api3.org/moonbeam) changes.
- Updated old links and tutorial smart contracts for Airnode, dAPIs and QRNG.
- Added newer QRNG Providers.
- Updated old screenshots related to API3 Market

### Checklist

- [ ] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Comment by eshaben** [comment]
@vanshwassan hey! thank you so much for opening this PR. I've had it on my backlog to update these docs! I'll review this shortly 🙂

File: builders/integrations/oracles/api3.md [hunk 1]
```diff
@@ -136,96 +142,47 @@ The data feeds are continuously updated by [first-party oracles](https://docs.ap
...
 
 !!! note
-    If a proxy is already deployed for a self-funded dAPI, the dApp can read the dAPI without having to deploy a proxy contract. They do this by using the address of the already deployed proxy contract which will be visible on the API3 Market.
-
-If you are deploying a proxy contract during the funding process, clicking on the **Get proxy** button will initiate a transaction to your wallet that will deploy a proxy contract.
-
-![API3 Remix deploy](/images/builders/integrations/oracles/api3/api3-8.webp)
-
-Once the transaction is broadcasted & confirmed on the blockchain, the proxy contract address will be shown on the UI.
+    If a dAPI is already activated, make sure to check the expiration date and update parameters. You can update the parameters and extend the subscription by purchasing a new configuration.
```
Comment (eshaben) [inline builders/integrations/oracles/api3.md:171]: How can I tell if the dAPI is already activated?

Comment (eshaben) [reply builders/integrations/oracles/api3.md:171]: I was checking out the testnet ETH/USD feed, and it showed the following:

Screenshot 2024-06-11 at 10 13 39 PM

Then I clicked purchase new plan, connected my wallet, clicked pruchase ETH/USD and signed the transaction. The transaction successfully went through, but nothing really changed on the UI. It shows the same as the above screenshot.

File: builders/integrations/oracles/api3.md [hunk 2]
```diff
@@ -136,96 +142,47 @@ The data feeds are continuously updated by [first-party oracles](https://docs.ap
...
+The [API3 Market](https://market.api3.org/moonbeam) lets users access dAPIs on both [Moonbeam Mainnet](https://market.api3.org/moonbeam) and [Testnet](https://market.api3.org/moonbeam-testnet).
 
-The process for accessing self-funded data feeds is as follows:
+#### Exploring, selecting and configuring your dAPI {: #explore-a-dapi }
 
-1. Explore the API3 Market and select a dAPI
-2. Fund a sponsor wallet
-3. Deploy a proxy contract to acess the data feed
-4. Read data from the dAPI
+The [API3 Market](https://market.api3.org/moonbeam) provides a list of all the dAPIs available across multiple chains including testnets. You can filter the list by mainnet or testnet chains. After selecting the chain, you can now search for a specific dAPI by name. Once selected, you will land on the details page (eg ETH/USD on Moonbeam Testnet) where you can find more information about the dAPI.
```
Comment (eshaben) [inline builders/integrations/oracles/api3.md:155]: Just noticed that the logo for Moonbeam is our old one. Can you please update it to our new logo, which can be found here: https://guidelines.moonbeam.foundation/logo

Is it possible to also change "Moonbeam testnet" to "Moonbase Alpha"? And the amounts for Moonbase are shown in GLMR, can that be updated to DEV? That's what we refer to the testnet token as and it's also what we instruct users to use when configuring their wallets

File: builders/integrations/oracles/api3.md [hunk 3]
```diff
@@ -136,96 +142,47 @@ The data feeds are continuously updated by [first-party oracles](https://docs.ap
...
 
-### Access Managed dAPIs {: #access-managed-dapis }
+After selecting the dAPI and the configuration, you will be presented with an option to purchase the dAPI and activate it. Make sure to check the time and amount of the subscription. If everything looks good, click on **Purchase**.
 
-If you are trying to access managed dAPIs, once you have selected your dAPI, you will then be presented with the option to choose from either **Managed** or **Self-funded**. Select **Managed dAPIs**.
+![API3 Market 2](/images/builders/integrations/oracles/api3/dapi-2.png)
 
-Managed dAPIs give you the option to configure the dAPI's [devation threshold](https://docs.api3.org/reference/dapis/understand/deviations.html){target=\_blank} and [heartbeat](https://docs.api3.org/reference/dapis/understand/deviations.html#heartbeat){target=\_blank}. For managed dAPIs, you will have the following options to choose from:
+You can then connect your wallet and confirm the transaction. Once it's confirmed, you will be able to see the updated configuration for the dAPI.
```
Comment (eshaben) [inline builders/integrations/oracles/api3.md:177]: Updated configuration? So does a free configuration exist? Is that why I am getting confused haha

File: builders/integrations/oracles/api3.md [hunk 4]
```diff
@@ -265,53 +224,68 @@ The example contract contains two functions:
 - `setProxy()` - used to set the address of the dAPI Proxy Contract
```
Comment (eshaben) [inline builders/integrations/oracles/api3.md:224]: This should be updated to `setProxyAddress`

File: builders/integrations/oracles/api3.md [hunk 5]
```diff
@@ -136,96 +142,47 @@ The data feeds are continuously updated by [first-party oracles](https://docs.ap
...
-
-#### Managed dAPIs {: #managed-dapis }
-
-[Managed dAPIs](https://docs.api3.org/reference/dapis/understand/managed.html) are sourced directly from multiple [first-party](https://docs.api3.org/explore/airnode/why-first-party-oracles.html){target=\_blank} data providers running an Airnode and aggregated using Airnode's signed data using a median function. The gas costs and availability of managed dAPIs are managed by the [API3 DAO](https://docs.api3.org/explore/dao-members){target=\_blank}.
-
-You can read more about [managed dAPIs on API3's documentation site](https://docs.api3.org/reference/dapis/understand/managed.html){target=\_blank}.
+### Subscribing to dAPIs {: #subscribing-to-dapis }
 
-### Access Self-Funded dAPIs {: #access-self-funded-dapis }
+The [API3 Market](https://market.api3.org/moonbeam) lets users access dAPIs on both [Moonbeam Mainnet](https://market.api3.org/moonbeam) and [Testnet](https://market.api3.org/moonbeam-testnet).
```
Comment (eshaben) [inline builders/integrations/oracles/api3.md:151]: Moonriver is supported as well, right?

File: builders/integrations/oracles/api3.md (hunk 1 above)
Comment (vanshwassan) [reply builders/integrations/oracles/api3.md:171]: You purchased the dAPI for the same parameters. You can purchase a better configuration for it. (For example: 0.25% deviation and 24hr heartbeat).

Comment (eshaben) [reply builders/integrations/oracles/api3.md:171]: Haha ah ok... makes sense 😂 thanks!

**Comment by eshaben** [comment]
@vanshwassan So we'll go through the same process as last time. I'll merge this into a local branch, make minor edits so it conforms with our standards, and then it'll go through review from our team before merging and publishing. I'll let you know once we've published the docs!

**Pull request merged**